sdk/core/Azure.Core/tests/ClientDiagnosticsTests.Net50.cs

```diff
@@ -99,6 +99,75 @@ public void StartsActivitySourceActivity()
             }
         }
 
+        [Theory]
+        [TestCase(0)]
+        [TestCase(2)]
```

Comment:
I think you can change these to the actual  `ActivityKind` values since they are constant

Reply:
unfortunately no, since ActivityKind is an internal type and can't be used in the public test

Reply:
Is it not this type? https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.activitykind?view=net-6.0

Reply:
Make use of new